docs/cep/error-handling/error-handling-wait-and-retry.md

```diff
@@ -5,54 +5,38 @@ title: Wait and Retry Example
 
 # Error Handling: Wait and Retry Example
 
-This example shows how errors are handled at the sink level with `wait and retry` mode.
+This section demonstrates the implementation of the `OnError.action='stream'` property in a wait and retry error handling strategy at the sink level.
```

Comment:
@diana-macrometa - Might be the annotation should be `OnError.action='wait'`?

Reply:
@diana-macrometa - @StoyanTc is correct. Since this section talks about `Wait and Retry Example` the action should be `wait`.